docs/docs/reference/Vectors.md

```diff
@@ -118,29 +118,74 @@ Optional parameters are:
 * `EF_CONSTRUCTION` - Number of maximum allowed potential outgoing edges candidates for each node in the graph, during the graph building. Default is 200.
 
 * `EF_RUNTIME` - Number of maximum top candidates to hold during the KNN search. 
-Higher values of `EF_RUNTIME` lead to more accurate results at the expense of a longer runtime. Defaul is 10.
+Higher values of `EF_RUNTIME` lead to more accurate results at the expense of a longer runtime. Default is 10.
+
+* `EPSILON` - Relative factor that sets the boundaries in which a range query may search for candidates. That is, vector candidates whose distance from the query vector is `radius*(1 + EPSILON)` are potentially scanned, allowing more extensive search and more accurate results (on the expense of runtime). Default is 0.01.   
 
 **Example**
 
 ```
 FT.CREATE my_index2 
 SCHEMA vector_field VECTOR 
 HNSW 
-14 
+16 
 TYPE FLOAT64 
 DIM 128 
 DISTANCE_METRIC L2 
 INITIAL_CAP 1000000 
 M 40 
 EF_CONSTRUCTION 250 
 EF_RUNTIME 20
+EPSILON 0.8
+```
+
+## Indexing vectors
+
+### Storing vectors in Hash
+Storing vectors in Redis Hashes is done by sending a binary blob of the vector data. A common way of doing so is by using python numpy with redis-py client:
+```py
+import numpy as np
+from redis import Redis
+
+redis_conn = Redis(host = 'localhost', port = 6379)
+vector_field = "vector"
+dim = 128
+
+# Store a blob of a random vector of type float32 under a field named 'vector' in Redis hash.
+np_vector = np.random.rand(dim).astype(np.float32)
+redis_conn.hset('key', mapping = {vector_field: np_vector.tobytes()})
+```
+Note that vector blob size must match the vector field dimension and type specified in the schema, otherwise the indexing will fail in the background.  
```

Comment:
```suggestion
Note that the vector blob size must match the vector field dimension and type specified in the schema, otherwise the indexing will fail in the background.  

```